docs/openapi/components/schemas/common/AgGradeInspectionDefect.yml

```diff
@@ -0,0 +1,61 @@
+$linkedData:
+  term: AgGradeInspectionDefect
+  '@id': https://w3id.org/traceability#AgGradeInspectionDefect
+title: Agriculture Grade Inspection Defect
+description: Information on a type of offsize / defect observed.
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - AgGradeInspectionDefect
+      - type: string
+        const:
+          - AgGradeInspectionDefect
+  offsizeDefect:
+    title: Offsize / Defect
+    description: Brief description of this offsize / defect.
+    type: string
+    $linkedData:
+      term: offsizeDefect
+      '@id': https://schema.org/description
+  averageDefects:
+    title: Average Defects
+    description: Percent of sample items with defects for the grade standards.
+    type: number
+    $linkedData:
+      term: averageDefects
+      '@id': https://schema.org/Number
```

Comment:
I think you are confusing id and type.

Reply:
Good to know. So, @id should never refer to a schema.org type? I chose Number because I couldn't find a better property on schema.org (like "percentage"). Do you think schema.org/value would work?

Reply:
@rhofvendahl, `https://qudt.org/vocab/unit/PERCENT` might be a better option for expressing percentage. 

Even better would be something expressing not just the data type, but this particular term. (Related: https://github.com/w3c-ccg/traceability-vocab/issues/230). I would encourage you taking a good look at GS1 EPCIS if there's a suitable definition you can point at.

Reply:
Thanks for the tip!

After poking through https://www.gs1.org/voc/ I still haven't find anything relating to percent of items with defects / that are unsuitable for some purpose. qudt.org/vocab/unit/PERCENT definitely seems more appropriate though.